tools/quickstart/quickstart/validator_account.py

```diff
@@ -171,3 +175,38 @@ def get_author_address(base_dir) -> str:
             return author_address_file.read()
     except FileNotFoundError:
         return get_validator_address(base_dir)
+
+
+def handle_legacy_validator_key_file(base_dir, chain_dir):
```

Comment:
The check and message is to strinct. It is perfectly fine to have more then one keystore file, just one has to be named account.json

so the check should be:
`if not validator_key_file_exist and legacy_file exist:
     if key_file_paths > 1:
           print("bla bla can not handle upgrading from one than more keyfiles, please rename one to account.json')
     else:
         rename
```

Reply:
So you'd say it is fine if there are two keys in this directory and we just ignore arbitrarily one? Wouldn't that be confusing to the user?

Reply:
I dont know what you mean by we ignore the other one. We need an account.json file, because that is the one that will be used by the bridge. So if there is account.json file we are already good.